test/powershell/Modules/Microsoft.PowerShell.Utility/typedata.tests.ps1

```diff
@@ -1,19 +1,23 @@
 ﻿Describe "TestData cmdlets" -Tags "CI" {
     Context "Get-TypeData" {
         It "System.Array" {
-            (Get-TypeData System.Array).TypeName | Should Be System.Array
+            Get-TypeData System.Array | Should Be $null
```

Comment:
@PetSerAl Yes. I made this change with the online editor. But I'm going to squash and merge, so it should be fine.